handler/handler.go

```diff
@@ -67,6 +71,25 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, event *models.Cont
 	}
 
 	log.Info(ctx, "event successfully handled", logData)
-
 	return nil
 }
+
+// incoming keywords validation
+func ValidateKeywords(keywords []string) []string {
+
+	var strArray []string
+	validKeywords := make([]string, 0)
+
+	for i := range keywords {
+		strArray = strings.Split(keywords[i], ",")
+		for j := range strArray {
+			validKeywords = append(validKeywords, strArray[j])
```

Comment:
Not sure what the requirements are but should we `trim` the keywords to remove any leading/trailing whitespace?

Reply:
good one, test included. Thanks